assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -264,16 +276,35 @@ function MediaPane(props) {
               onChange={onFilter}
               options={FILTERS}
               placement={Placement.BOTTOM_START}
+              widthFitContent
             />
-            <Primary onClick={openMediaPicker}>
-              {__('Upload', 'web-stories')}
-            </Primary>
+            {isSearching && media.length && (
+              <SearchCount>
+                {sprintf(
+                  /* translators: %d: number of results. */
+                  _n(
+                    '%d result found',
+                    '%d results found',
+                    media.length,
+                    'web-stories'
+                  ),
+                  media.length
```

Comment:
This number is not correct. When you search, it does a request to REST API to get the first 100 items, it then loads more on scroll. If you had say 150 results, this would result in 100, as it would only get the result of the first request. 

There is a field in the result called `X-WP-Total` that gives the full total of result. 

<img width="453" alt="Screenshot 2021-01-04 at 10 35 53" src="https://user-images.githubusercontent.com/237508/103526384-bd114600-4e78-11eb-8f03-1cbd1a69b081.png">

Reply:
Good catch!

Sounds like we'd need to do `const totalItems = parseInt(headers['X-WP-Total']);` here and pass it along as well:

https://github.com/google/web-stories-wp/blob/49adc48243db2d149c8623119ae63e838bbe971d/assets/src/edit-story/app/media/local/useContextValueProvider.js#L90-L102

Reply:
Thanks for the guide, updated!